recipes/hitac/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
It would be good to ask the upstream author (or is that you?) to add a proper `setup.py` file to their program. In that way, the tool could be installed with a simple `pip install` and this build script won’t be necessary anymore.

Reply:
Yep, it is me. Done!